### PR TITLE
Remove 'live-like' contact information from sample data

### PR DESCRIPTION
**Pull request by RahulJha-mastek**
Story [AB#14239](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14239)
Task [AB#14445](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14445)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql [hunk 1]
```diff
@@ -68,28 +68,28 @@ BEGIN
...
+    (NEWID(),'100013', 'Jeannie',  'Houdini',   'pahmed@test.test',             '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100014', 'Marisa',   'Fletcher',  'marflet@test.test',            '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100015', 'Myra',     'Samosa',    'myra280s@test.test',           '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100016', 'Renato',   'Samosa',    'xpamela.gomes.pa2@test.test',  '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100017', 'Angel',    'Teabag',    'nael@test.test',               '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100018', 'Lindsay',  'Burton',    'linburt@test.test',            '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100019', 'Xavier',   'Eggbert',   'esamuel@eggs.test',            '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'100020', 'Mario',    'Rocks',     'noulmes14b@mario.test',        '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'99998',  'Aimee',    'Mcdonald', 'mcaimee@test.test',             '+44 1234 567891',  @now, @emptyGuid),
+    (NEWID(),'99999',  'Sophia',   'Rojas',    'rosa@test.test',                '+44 1234 567891',  @now, @emptyGuid);
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql:93]: I think we probably need to tweak some of these names a little more. 'Ashley Banjo', even though it's a slightly unusual name, is a real person, for example.

Feel free to reuse the names used in `InsertSolutions`.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql [hunk 2]
```diff
@@ -68,28 +68,28 @@ BEGIN
 
     INSERT INTO [dbo].[SupplierContact] ([Id], [SupplierId],[FirstName],[LastName],[Email],[PhoneNumber],[LastUpdated],[LastUpdatedBy])
     VALUES
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql:70]: While you making changes to this file could you remove the square brackets and make sure there's a space after each comma in the `INSERT` statement?

**Review by bjss-chris — CHANGES_REQUESTED** [review]

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql (hunk 1 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql]: I have made some changes and also reused names from `InsertSolutions`

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql (hunk 2 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql]: Updated format of Insert statement

**Pull request merged**